libs/network/esp8266/jswrap_esp8266_network.c

```diff
@@ -753,6 +753,28 @@ JsVar *jswrap_ESP8266_wifi_getStatus() {
   return jsWiFiStatus;
 }
 
+/**
+ * Get the ip info for the given interface.  The interfaces are:
+ * * 0 - Station
+ * * 1 - Access Point
+ */
+static JsVar *getIPInfo(int interface) {
+  struct ip_info info;
+  wifi_get_ip_info(interface, &info);
+
+  JsVar *jsIpInfo = jspNewObject(NULL, "IPInfo");
```

Comment:
Oops. this is the new code. I guess you haven't defined a class called `IPInfo` for this, and even if you did, it'd be wasting some RAM.

Just use `= jsvNewWithFlags(JSV_OBJECT)` like everything else does.